packages/mc-html-template/webpack.js

```diff
@@ -0,0 +1,25 @@
+const generateTemplate = require('./lib/generate');
+
+module.exports = templateParams => {
```

Comment:
This is the extracted part related to the html webpack plugin, which only does some mapping